### DISCUSSION TITLE
new dota item able to remove opposite player item in late game

### DISCUSSION THREAD

**Discussion by starlight32** [ValveSoftware/Dota2-Gameplay#12354] [Ideas]
to enable more option of survival in late game

**Comment by commentator24** [comment]
yes we need a red card in dota. the match may resume only after the referee has blown his whistle

The referee shows the red card if the dota player:

• commits a severe foul.
• becomes violent.
• insults somebody.
• prevents the opponent from clearly scoring towers with a so-called "professional foul" (which is a deliberate act of foul backdoor play).
😆

**Comment by run3unblockedaz** [comment]
Your forum contributions are extremely insightful, and we would appreciate it if you left feedback on the <a href="https://lol-beans.com/">**lol beans**</a> website so that we could continue to develop it. Additionally, we would like to honor you for your insights.
